README.md

```diff
@@ -1,307 +1,29 @@
-        }
-      ]
-    }
-  }
-}
-'
-{
-  "id" : "4eOWgYIBAYdMiGHcxmKH",
-  "name" : "my-connector",
-  "api_key" : "zIJThFg9TO6uaYVy57TFSA",
-  "encoded" : "NGVPV2dZSUJBWWRNaUdIY3htS0g6eklKVGhGZzlUTzZ1YVlWeTU3VEZTQQ=="
-}
-```
-
-And then use the `encoded` value:
-
-```
-make run-docker API_KEY="NGVPV2dZSUJBWWRNaUdIY3htS0g6eklKVGhGZzlUTzZ1YVlWeTU3VEZTQQ=="
-```
-
-## Operating the connector
-
-As stated above, the connector will be operated by an administrative user from within Kibana.
-
-- The connector operator sets the sync schedule in Kibana, and Kibana writes this to the connector's document in Elasticsearch. So the connector has to read this document to determine when to next sync.
-- The connector operator can see the state of the connector within Kibana. So the connector must regularly write its state back to the document, where Kibana can read it.
-
-### Configuring the connector in Kibana
-
-- In Kibana UI, navigate to `Enterprise Search` -
-  `Content` - `Elasticsearch indices`.
-- Click on the index that you created for the connector while [implementing the connector protocol](#implementing-the-connector-protocol).
-- Go to the tab `Configuration` and make sure the following message is displayed:
-
-```text
-Your connector <your-service-name> has connected to Enterprise Search successfully.
-```
-
-- If needed, set the values of configurable fields in the `Configuration` tab, using the `Edit configuration` button.
-- Use the `Set schedule and sync` on the same tab to set the sync schedule and run the sync for the first time.
-
-### Testing the integration
-
-If the connector synchronizes successfully, the documents should appear in `Enterprise Search
-Content - Elasticsearch indices - your-index-name` section, on the `Documents` tab.
-
-If the connector didn't specify any configurable fields, it will go into the `connected` status and will try to sync, without requiring any additional configuration via the Kibana UI.
-
-If the connector fails to synchronize, it will change its status to `error`.
-
-## Moving to production
-
-After you've implemented everything correctly, you're ready to go to Production. Before you do this, make sure the following topics are covered:
-
-- No sensitive values are committed to source code repositories.
-- No unsecured values are exposed via configurable fields.
-- The connector is properly authenticated to the third-party data source.
-- The connector implements the connector protocol correctly and does appropriate error handling.
-
-## Contribute to the repository 🚀
-
-We welcome contributors to the project. Before you begin, please read the [Connectors Contributor's Guide](docs/CONTRIBUTING.md).
+- [Contribute to the repository](docs/CONTRIBUTING.md)
 
 ## Other guides
 
 - [Code of Conduct](https://www.elastic.co/community/codeofconduct)
-- [Getting Support](docs/SUPPORT.md)
+- [Configuration](docs/CONFIG.md)
 - [Releasing](docs/RELEASING.md)
-- [Developer guide](docs/DEVELOPING.md)
```

Comment:
Why was this link removed?

EDIT: I see, because it's linked above? I kinda like the idea of us having an index here that links to all our other supporting guides, even if they're mentioned elsewhere. But this is a loosely held opinion.